election/election.go

```diff
@@ -861,7 +859,7 @@ func (e *node) send(req interface{}) chan RPCResponse {
 
 // randomDuration returns a value that is between the minDur and 2x minDur.
 func randomDuration(minDur time.Duration) time.Duration {
-	return minDur + time.Duration(rand.Int63())%minDur
+	return minDur + time.Duration(rand.Int63())%minDur // #nosec
```

Comment:
It stinks that nosec doesn't follow the `nolint` syntax.  However, replacing with `//nolint` removes context of what the problem is.  So, I'd have to explain it with more words in the comment.

btw, I did add a comment to this since you reviewed.

Reply:
I meant `//nolint:gosec // Reason`...

Reply:
Correction, it does allow `//nolint:gosec`.  I must've typed it wrong before.  Fixed.